benchmark/Main.hs

```diff
@@ -13,19 +14,23 @@ import qualified Data.Text.IO as TIO
 main :: IO ()
 main = do
     txt <- TIO.readFile "./benchmark/benchmark.toml"
-    Right htomlVal <- pure $ Htoml.parse txt
     Right tomlandVal <- pure $ Tomland.parse txt
+    Right htomlVal <- pure $ Htoml.parse txt
+    Right htomlMegaVal <- pure $ HtomlM.parse "log" txt
     defaultMain
         [ bgroup "Parse"
-            [ bench "tomland" $ nf Tomland.parse txt
-            , bench "htoml"   $ nf Htoml.parse txt
+            [ bench "tomland"          $ nf Tomland.parse txt
+            , bench "htoml"            $ nf Htoml.parse txt
+            , bench "htoml-megaparsec" $ nf (HtomlM.parse "log") txt
```

Comment:
Let's move this `"log"` into `HtomlM` module to have consistent look of benchmarks.